python/lsst/daf/butler/timespan_database_representation.py

```diff
@@ -66,7 +66,7 @@ class _SpecialTimespanBound(enum.Enum):
     """
 
 
-TimespanBound = Union[astropy.time.Time, _SpecialTimespanBound, None]
+TimespanBound: TypeAlias = astropy.time.Time | _SpecialTimespanBound | None
```

Comment:
I'm not sure why this code is copied from `_timespan.py`.

Reply:
It was just a mistake made when refactoring one module into two modules.  We should remove it and import instead it if it's actually used here.